pkg/testing/integration/program.go

```diff
@@ -573,7 +573,8 @@ func GetLogs(
 	stackInfo RuntimeValidationStackInfo,
 	query operations.LogQuery) *[]operations.LogEntry {
 
-	snap, err := stack.DeserializeDeploymentV3(*stackInfo.Deployment, stack.DefaultSecretsProvider)
+	snap, err := stack.DeserializeDeploymentV3(context.Background(),
```

Comment:
Assuming safe to do context.Background() here but could not figure out where this func is used, in testing?